src/I18n/Time.php

```diff
@@ -856,10 +856,10 @@ public static function parseTime($time, $format = null)
     /**
      * Convenience method for getting the remaining time from a given time.
      *
-     * @param DateTime $datetime The date to get the remaining time from.
-     * @return DateInterval|bool The DateInterval object representing the difference between the two dates or FALSE on failure.
+     * @param \DateTime $datetime The date to get the remaining time from.
+     * @return \DateInterval|bool The DateInterval object representing the difference between the two dates or FALSE on failure.
      */
-    public static function fromNow($datetime)
+    public static function fromNow(\DateTime $datetime)
```

Comment:
OK, can I assume that this does apply generally then? ie not only in function/method signatures, but everywhere in the code (`new Abc()`, Abc::xyz(), `catch (Abc)`, ...)?